pkg/util/mount/mount_linux_test.go

```diff
@@ -1395,6 +1462,15 @@ func TestSafeOpen(t *testing.T) {
 		if err != nil {
 			t.Fatalf(err.Error())
 		}
+
+		// If base path is too long, creation of unix sockets will fail.
+		if len(base) > 75 {
```

Comment:
do you still need this?

Reply:
No I don't. I am waiting for master PR to get merged, I will simply cherry-pick that one verbatim rather than creating new commit.